app/assets/stylesheets/petitions/admin/_meta.scss

```diff
@@ -0,0 +1,7 @@
+.admin-petition-meta {
```

Comment:
FYI: In other admin scss files we are using namespacing to make it doubly specific. This is OK though cause it is clearly an admin class in an admin folder